templates/base_clean.html

```diff
@@ -36,15 +36,14 @@
     <div class="container-fluid">
       <div class="navbar-header">
         <a class="navbar-brand" href="{% if user.is_authenticated %}{% url 'url_group_workboard' group_name='gadzarts' %}{% endif %}">
-           <span class="logo"></span>
```

Comment:
Hmm ?

Reply:
The logo on the navbar is not used in base clean template